android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -1461,6 +1461,10 @@ protected void registerForTouch(final View touchable)
 		boolean clickable = true;
 		if (proxy.hasProperty(TiC.PROPERTY_TOUCH_ENABLED)) {
 			clickable = TiConvert.toBoolean(proxy.getProperty(TiC.PROPERTY_TOUCH_ENABLED), true);
+			//If touchEnabled is specifically set, enable or disable touch sound
+			touchable.setSoundEffectsEnabled(clickable);
+		} else {
+			touchable.setSoundEffectsEnabled(false);
```

Comment:
By making this change, it would change the default (native) behavior. I'm good with deactivating the sound when touches are not enabled, but it shouldn't change the default behavior. What do you think?